io/js/src/main/scala/fs2/io/ioplatform.scala

```diff
@@ -199,9 +199,11 @@ private[fs2] trait ioplatform {
       in: Stream[F, Byte]
   )(implicit F: Async[F]): Resource[F, (Duplex, Stream[F, Byte])] =
     for {
-      dispatcher <- Dispatcher[F]
+      readDispatcher <- Dispatcher.sequential[F]
+      writeDispatcher <- Dispatcher.sequential[F]
+      errorDispatcher <- Dispatcher.sequential[F]
```

Comment:
read/write are sequential and error is one-off, but they need to be independent, so they need their own dispatchers.